### PR TITLE
common FTA files

### PR DESCRIPTION
**Pull request by noronhaa**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by sullivangareth — APPROVED** [review]

File: src/functionalTest/java/uk/gov/hmcts/reform/cpo/befta/CaseIdGenerator.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.cpo.befta;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+import java.util.Date;
+
+public class CaseIdGenerator {
```
Comment (mattnayler) [inline src/functionalTest/java/uk/gov/hmcts/reform/cpo/befta/CaseIdGenerator.java:7]: DO you also need to add your changes to the [CasePaymentOrdersTestAutomationAdapter.java](https://github.com/hmcts/cpo-case-payment-orders-api/blob/CPO-FTA/src/functionalTest/java/uk/gov/hmcts/reform/cpo/befta/CasePaymentOrdersTestAutomationAdapter.java) to make this useable to others?

File: src/functionalTest/resources/features/common/Base_CPO_Payload_Data.td.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "_guid_" : "Base_CPO_Data",
```
Comment (mattnayler) [inline src/functionalTest/resources/features/common/Base_CPO_Payload_Data.td.json:2]: sorry: pet hate when GUIDs and file names don't match.

File: src/functionalTest/resources/features/common/response/Common_204_Response.td.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "_guid_" : "Common_204_Response2",
```
Comment (mattnayler) [inline src/functionalTest/resources/features/common/response/Common_204_Response.td.json:2]: sorry: pet hate when GUIDs and file names don't match.

**Review by mattnayler — CHANGES_REQUESTED** [review]
Good additions but a few minor comments and one big one.  [Can you clear the checkstyle issues](https://build.platform.hmcts.net/job/HMCTS_CDM/job/cpo-case-payment-orders-api/job/PR-59/2/artifact/build/reports/checkstyle/functionalTest.html)

File: src/functionalTest/java/uk/gov/hmcts/reform/cpo/befta/CaseIdGenerator.java (hunk 1 above)
Comment (noronhaa) [reply src/functionalTest/java/uk/gov/hmcts/reform/cpo/befta/CaseIdGenerator.java:7]: ahh good spot, had forgot to port across changes from that class onto this branch

File: src/functionalTest/resources/features/common/Base_CPO_Payload_Data.td.json (hunk 2 above)
Comment (noronhaa) [reply src/functionalTest/resources/features/common/Base_CPO_Payload_Data.td.json:2]: sure thing

**Review by mattnayler — APPROVED** [review]
LGTM 👍

**Pull request merged**